pcsx2/USB/usb-eyetoy/cam-windows.cpp

```diff
@@ -230,12 +230,15 @@ namespace usb_eyetoy
 									(pmtConfig->cbFormat >= sizeof(VIDEOINFOHEADER)) &&
 									(pmtConfig->pbFormat != nullptr))
 								{
-
 									VIDEOINFOHEADER* pVih = (VIDEOINFOHEADER*)pmtConfig->pbFormat;
 									pVih->bmiHeader.biWidth = frame_width;
 									pVih->bmiHeader.biHeight = frame_height;
 									pVih->bmiHeader.biSizeImage = DIBSIZE(pVih->bmiHeader);
 									hr = pSourceConfig->SetFormat(pmtConfig);
+									if (FAILED(hr))
+									{
+										Console.Warning("Camera: SetFormat err : %x", hr);
```

Comment:
Might be nice addendum to mention that the category is `USB:`

Reply:
"USB:" isn't used anywhere in this file